examples/ed25519/src/ed25519.rs

```diff
@@ -0,0 +1,909 @@
+//! This crate gives a specification for the EdDSA signature scheme ed25519.
+//! As there is no proper consensus for the verification algorithm, it has been implemented in multiple ways.
+//!
+//! The different implementations and specifications differ mainly in three different ways:
+//! - Cofactored/cofactorless verification -- (Using \[8\]\[S\]B = \[8\]R + \[8\]\[H(R, A, msg)\]A or \[S\]B = R + \[H(R, A, msg)\]A as the verification check).
+//! - Accepting/rejecting non-canonical encodings of points.
+//! - Accepting/rejecting small-order points.
+//! 
+//! Batch verification has also been implemented, but one should be careful when using cofactorless batch verification, as on some inputs
```

Comment:
Shouldn't this spec make sure that cofactorless batch verification isn't happening?